.github/CODEOWNERS

```diff
@@ -4,8 +4,9 @@
 *   @hCaptcha/react-reviewers
 
 # Javascript Owners
-*.js    @hCaptcha/react-reviewers
```

Comment:
Lets have both for these, I believe you can separate by comma or just duplicate line

Reply:
moved this change to separate [PR](https://github.com/hCaptcha/hcaptcha-loader/pull/21)